### PR TITLE
WIP: Add etcd framework

### PR DESCRIPTION
**Pull request by sttts**
TODO:

- [x] add proper health check (now it's just /stats)
- [x] get health check to be considered by dcos ui
- [x] fix icons, not visible in dcos ui at the moment
- [x] add support for webui uri

### CODE REVIEW

**Comment by jdef** [comment]
looks like a kubernetes project file bled into this commit unintentionally

**Comment by sttts** [comment]
@jdef you are right. this does not belong there.

File: repo/meta/index.json [hunk 1]
```diff
@@ -27,6 +27,20 @@
...
     },
     {
+      "currentVersion":"v0.0.1",
+      "description":"Manage a cluster of etcd.",
+      "framework":true,
+      "name":"etcd",
+      "tags":[
+        "mesosphere",
+        "framework",
+        "etcd"
```
Comment (jdef) [inline repo/meta/index.json]: pretty sure that this `etcd` is redundant since it's already the package name

**Comment by ConnorDoyle** [comment]
:balloon: super excited for this!  Thanks!

**Comment by kensipe** [comment]
WIP and experimental work should go into the multiverse first... could you submit a PR against https://github.com/mesosphere/multiverse/ on the master?

**Comment by sttts** [comment]
@kensipe we would leave it here unmerged until it's ready anyway. It will be a dependency soon for kubernetes which lives here in universe.

**Comment by kensipe** [comment]
@sttts kubernetes is moving to the multiverse.   we are in the process of maturing our universe repo policies as well.    Should have it out this week.

**Comment by sttts** [comment]
Superseded by https://github.com/mesosphere/multiverse/pull/7. Closing.

**Pull request closed without merging**